etc/openstack_deploy/group_vars/all/osa.yml

```diff
@@ -96,3 +96,15 @@ haproxy_extra_services:
 
 # Define the distro version globally
 repo_build_os_distro_version: "{{ (ansible_distribution | lower) | replace(' ', '_') }}-{{ ansible_distribution_version.split('.')[:2] | join('.') }}-{{ ansible_architecture | lower }}"
+
+# remove once https://review.openstack.org/#/c/573822/ merges
```

Comment:
do we have a text blob like `TODO` for items needing cleanup?  is there a jira issue for the removal?